### PR TITLE
Added `isOpen(id)` public method.

### LINKED ISSUE #216: Check if dialog is open?
Hello!

Is it possible to check if dialog is still open?

It looks like [returned object](https://github.com/likeastore/ngDialog#returns) doesn't provide such facilities, or am I wrong?

Thank you!

### PR DESCRIPTION
**Pull request by kasimoglou**
This way user can check anytime whether a dialog he has previously created,
is still open.
`open()` function returns an object that contains the `id` of the newly created dialog.
User can check if this dialog is open by: ngDialog.isOpen(new_dialog_object.id);

Resolves #216

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 .DS_Store
 node_modules
 bower_components
-*.log
\ No newline at end of file
+*.log
+/nbproject
```
Comment (voronianski) [inline .gitignore]: ?

Comment (kasimoglou) [inline .gitignore]: My mistake! Fixed!

**Comment by voronianski** [comment]
@kasimoglou thanks! It will be nice to update README with few words about that method ;)

**Comment by kasimoglou** [comment]
@voronianski Thank you for accepting my request! I will update README too! :)

**Pull request merged**